### ISSUE TITLE
Links like "Learn More", "Read More" should provide more context

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1060]
FROM a11y spreadsheet: 

When authoring labels, set user expectations about the purpose of each link, control, and button, so users can decide whether they want to follow it. All links, controls, and buttons should have comprehensible text or tooltip text, so users can identify its purpose, independent of context, such as when using the URL list from TalkBack local context menu. Uncomplicated text also helps people with cognitive disabilities navigate more easily. From the GAR document, here’s an example of what not to do. “Hyperlinked text that says “[click here] to learn more”. The text in brackets would be linked. Indistinguishable text makes it difficult for users to skim content by links and quickly understand the author’s intention.
Links with the same destination should have the same description, ideally the title of the destination, while links with different purposes and destinations should have different descriptions.

*Make sure the visible text on “learn more” links is specific and identifies what the users will learn more about.
Avoid using “Click here”, “here”, “more”, “read more”, “link to [link destination]”, or “info” and follow best practices to write descriptive link text.”*

**Comment by jgonzalezmsm** [comment]
I'm worried that this change will crowd the page. Accessibility concerns warranted; can you mock up a design that would maybe refine the spacing? It could look better if the images are all the same size and we have consistent spacing between the text blocks - maybe that will give space for the descriptive links underneath each text? Also should we standardize read more vs learn more? what's the difference

**Comment by benhammondmusic** [comment]
Standardizing "learn more" and "read more" was also addressed in the A11y spreadsheet but this rule seemed to supersede that. 

I think even super simple detail might be helpful; eg "Read Article" instead of "read more".
